docs/user/README.md

```diff
@@ -252,6 +252,18 @@ By using Telegraf, data points can be efficiently streamed into `modelardbd` fro
+
+| **Variable**                                     | **Default** | **Description**                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------------------------------|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MODELARDBD_PORT                                  | 9999        | The port of the Apache Arrow Flight Server.                                                                                                                                                                                                                                                                                                                |
+| MODELARDBD_UNCOMPRESSED_RESERVED_MEMORY_IN_BYTES | 512 MB      | Amount of memory to reserve for storing uncompressed data buffers.                                                                                                                                                                                                                                                                                         |
+| MODELARDBD_COMPRESSED_RESERVED_MEMORY_IN_BYTES   | 512 MB      | Amount of memory to reserve for storing compressed data buffers.                                                                                                                                                                                                                                                                                           |
+| MODELARDBD_TRANSFER_BATCH_SIZE_IN_BYTES          | 64 MB       | The number of bytes that are required before transferring a batch of  data to the remote object store.                                                                                                                                                                                                                                                     |
+| MODELARDBD_UNCOMPRESSED_DATA_BUFFER_CAPACITY     | 65536       | The capacity of each uncompressed data buffer as the number of elements in the buffer where each element is a timestamp and a value. Note that the resulting size of the buffer has to be a multiple of 64 bytes to avoid the actual capacity being larger than the requested due to internal alignment when allocating memory for the two array builders. |
```

Comment:
Maybe remove "...when allocating memory for the two array builders." as it seem to be too implementation specific for the user documentation.

Reply:
Fixed.